<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QFrame" name="inputsFrame">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>251</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="inputsLayout"/>
   </widget>
  </widget>
  <widget class="QFrame" name="buttonsFrame">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="removeSegmentButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addSegmentButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="accelerationLimitFrame">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="accLimInput">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="accLimLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Acceleration limit:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
